src/main/java/org/dependencytrack/integrations/defectdojo/DefectDojoClient.java

```diff
@@ -40,66 +51,74 @@ public class DefectDojoClient {
+        nameValuePairs.add(new BasicNameValuePair("close_old_findings", "true"));
+        nameValuePairs.add(new BasicNameValuePair("push_to_jira", "false"));
+        nameValuePairs.add(new BasicNameValuePair("scan_date", DATE_FORMAT.format(new Date())));
 
-        final HttpResponse<String> response = request
-                .header("accept", "application/json")
-                .header("Authorization", "Token " + token)
-                .field("file", findingsJson, "findings.json")
-                .field("engagement", engagementId)
-                .field("scan_type", "Dependency Track Finding Packaging Format (FPF) Export")
-                .field("verified", "true")
-                .field("active", "true")
-                .field("minimum_severity", "Info")
-                .field("close_old_findings", "true")
-                .field("push_to_jira", "false")
-                .field("scan_date", DATE_FORMAT.format(new Date()))
-                .asString();
-        if (response.getStatus() == 201) {
-            LOGGER.debug("Successfully uploaded findings to DefectDojo");
-        } else {
-            LOGGER.warn("DefectDojo Client did not receive expected response while attempting to upload "
-                    + "Dependency-Track findings. HTTP response code: "
-                    + response.getStatus() + " - " + response.getStatusText());
-            uploader.handleUnexpectedHttpResponse(LOGGER, request.getUrl(), response.getStatus(), response.getStatusText());
+        HttpEntity data = MultipartEntityBuilder.create().setMode(HttpMultipartMode.BROWSER_COMPATIBLE)
+                .addBinaryBody("file", findingsJson, ContentType.APPLICATION_JSON, "findings.json")
+                .build();
+        request.setEntity(data);
+        request.setEntity(new UrlEncodedFormEntity(nameValuePairs, StandardCharsets.UTF_8));
+        try {
+            CloseableHttpResponse response = HttpClientPool.getClient().execute(request);
+            if (response.getStatusLine().getStatusCode() == HttpStatus.SC_CREATED) {
+                LOGGER.debug("Successfully uploaded findings to DefectDojo");
+            } else {
+                LOGGER.warn("DefectDojo Client did not receive expected response while attempting to upload "
+                        + "Dependency-Track findings. HTTP response code: "
+                        + response.getStatusLine().getStatusCode() + " - " + response.getStatusLine().getReasonPhrase());
+                uploader.handleUnexpectedHttpResponse(LOGGER, request.getURI().toString(), response.getStatusLine().getStatusCode(), response.getStatusLine().getReasonPhrase());
+            }
+        } catch (IOException ex) {
+            LOGGER.error("Error while sending request from upload DT findings defectDojo Client" + ex.getMessage());
+            LOGGER.error("Error while sending request from upload DT findings defectDojo Client" + ex.getStackTrace());
```

Comment:
duplicated I think. It can be one log message with message and throwable cause

Reply:
removed this duplicate